bin/ci/az-publish.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+source bin/ditto.sh
+
+set -eo pipefail
+CONFIG=Debug
```

Comment:
What does this do?

Reply:
This line was copied from `Xamarin.UITest/nunit-extensions/nunit3/az-publish.sh`. Forgot to remove right away. Removed.